pydantic/utils.py

```diff
@@ -1,16 +1,10 @@
 import re
+import email_validator
```

Comment:
I think we should catch the `ImportError` if `email_validator` isn't installed and either:
* raise an error if `validate_email` is called
* or, fallback to `EMAIL_REGEX`

Reply:
You also need to correct import order.

Reply:
If we create a branch with `EMAIL_REGEX`, we'll have to create a branch in the unit tests, too. I don't think it is a good idea to consider an email address valid or not, depending on the modules installed.

Reply:
Ok, I still don't want to make `email_validator` an install requirement of pydantic, but I agree that different behaviour depending on whether the package is installed would confuse people. Therefore do something like

```py
 def validate_email(value) -> Tuple[str, str]:
    try:
        import email_validator
    except ImportError as e:
        raise ImportError('email_validator need to be installed to in order to use pydantic for email validation, please run `pip install email_validator`') from e
    ...
```